### PR TITLE
move to ES6, second try

### PR DESCRIPTION
**Pull request by fippo**
revisiting #825. Most of it was possible to salvage

### CODE REVIEW

File: src/js/adapter_factory.js [hunk 1]
```diff
@@ -5,49 +5,36 @@
...
   // will not appear.
-  // require('./utils').disableLog(false);
-
-  // Browser shims.
-  var chromeShim = require('./chrome/chrome_shim') || null;
-  var edgeShim = require('./edge/edge_shim') || null;
-  var firefoxShim = require('./firefox/firefox_shim') || null;
-  var safariShim = require('./safari/safari_shim') || null;
-  var commonShim = require('./common_shim') || null;
+  // utils.disableLog(false);
```
Comment (dagingaa) [inline src/js/adapter_factory.js:32]: Remove this?

**Review by dagingaa — APPROVED** [review]
Seems like most of this is just done by codemods, looks correct but hard to tell. I support this move!

File: src/js/adapter_factory.js (hunk 1 above)
Comment (fippo) [reply src/js/adapter_factory.js:32]: will do as part of "remove cruft"

**Comment by fippo** [comment]
now on to remote some cruft...

**Pull request merged**